### PR TITLE
Hidden Wifi Network

### PR DESCRIPTION
**Pull request by Aleyum**
- Added new field `hidden `and SDK Enum `NetworkType `to EnterpriseWifiConnectionRequest

### CODE REVIEW

File: Assets/MXR.SDK/Runtime/Types/WifiTypes.cs [hunk 1]
```diff
@@ -313,9 +313,13 @@ public class EnterpriseWifiConnectionRequest
         public string ssid;
         public string password;
         public string identity;
+        [DefaultValue("")]
```
Comment (adrenak) [inline Assets/MXR.SDK/Runtime/Types/WifiTypes.cs:316]: Shouldn't this be set to `[DefaultValue(false)]` instead? This way it gets serialized only when it's set to true

Comment (Aleyum) [reply Assets/MXR.SDK/Runtime/Types/WifiTypes.cs]: Addressed

**Review by johnmanagexr — COMMENTED** [review]
Looks good! Have you tested this?

**Comment by Aleyum** [comment]
@johnmanagexr yes.

**Review by johnmanagexr — APPROVED** [review]

**Pull request merged**